src/main/java/com/mercuryCyclists/Sales/entity/SaleEvent.java

```diff
@@ -0,0 +1,40 @@
+package com.mercuryCyclists.Sales.entity;
+
+public class SaleEvent {
```

Comment:
use an all args constructor annotation?

Reply:
I mean we can but currently uses default + setters